src/Response/SearchByFeatureResponse.php

```diff
@@ -20,7 +20,7 @@ private function __construct(array $uids)
     public static function fromArray(array $response): SearchByFeatureResponse
     {
         try {
-            $products = $response['products'];
+            $products = $response[0]['products'];
```

Comment:
what about `[1]`, `[2]` et al? What's their purpose? Can we safely assume that `[0]` is what we always need?

Reply:
Please ask WideEyes what the heck :)

Reply:
I've done that, I'm waiting for an answear.

Reply:
`answer`?